br/pkg/encryption/master_key/mem_backend_test.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2024 PingCAP, Inc. Licensed under Apache-2.0.
+
+package encryption
+
+import (
+	"bytes"
+	"context"
+	"testing"
+)
+
+func TestNewMemAesGcmBackend(t *testing.T) {
+	key := make([]byte, 32) // 256-bit key
+	_, err := NewMemAesGcmBackend(key)
+	if err != nil {
+		t.Fatalf("Failed to create MemAesGcmBackend: %v", err)
+	}
+
+	shortKey := make([]byte, 16)
+	_, err = NewMemAesGcmBackend(shortKey)
+	if err == nil {
+		t.Fatal("Expected error for short key, got nil")
```

Comment:
You may use [`testify/require`](https://pkg.go.dev/github.com/stretchr/testify/require) instead of explicitly check.

Reply:
ah good call, this test was first written, forgot to change it to use require